gradle-plugin/kubernetes/src/test/java/org/eclipse/jkube/gradle/plugin/task/TaskEnvironmentExtension.java

```diff
@@ -65,17 +67,35 @@ protected void before() throws Throwable {
   }
 
   @Override
-  protected void after() {
-    super.after();
+  public void afterEach(ExtensionContext context) throws Exception {
     defaultTaskMockedConstruction.close();
+    recursiveDelete(temporaryFolder);
```

Comment:
instead of manually deleting folder is it possible to use `deleteOnExit()` ?

Reply:
That's not working but the below ones are working

`    FileUtil.cleanDirectory()` and ` FileUtils.deleteDirectory()`

which one to use?

Reply:
I'm okay with any option as long as we don't manually do it.

Reply:
I think the original implementation was replicating what the `TemporaryFolder` rule did.
(`delete`,`tryDelete`,`recursiveDelete`)